docs/install_guides/installing-openvino-linux.md

```diff
@@ -33,6 +33,12 @@
   
   .. _Product Specifications: https://ark.intel.com/
 
+.. tab:: Software
+
+  * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+  * GCC* 7.5.0 (for Ubuntu* 18.04) or GCC* 9.3.0 (for Ubuntu* 20.04)
+  * `Python 3.6 - 3.9, 64-bit <https://www.python.org/downloads/windows/>`_
+
```

Comment:
@slyubimt do we need such info? Looks like it's not mandatory to have cmake.. not sure how to position others.